tools/devctr/Dockerfile

```diff
@@ -54,6 +54,8 @@ RUN apt-get update \
         flex \
         bison \
         bc \
+        # for cpu-template-helper
+        dmidecode \
```

Comment:
See my previous comment, maybe we can avoid this

Reply:
See https://github.com/firecracker-microvm/firecracker/pull/3654#discussion_r1190868106